### PR TITLE
add api test

### PR DESCRIPTION
**Pull request by deniseileen**
Add test for react component

### CODE REVIEW

File: spec/controllers/api/v1/sandwiches_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  describe "GET#index" do
+    before do
+      get :index
+
+    end
+
+
+
+    it "returns http success" do
```
Comment (LiannaNewman) [inline spec/controllers/api/v1/sandwiches_controller_spec.rb]: Watch your spacing. Remove line breaks from lines 12, 14, and 15. Also your ends on line 43 and 44 seem a bit scattered. On line 44 the `end` should only be indented 3 spaces to line up with `describe` on line 9. And on line 43, the `end` should be indented 5 spaces to line up with the `it` on line 21.

File: spec/controllers/api/v1/sandwiches_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      expect(response).to have_http_status(:success)
+    end
+
+    it "response with JSON body containing a list of Sandwiches" do
+      # binding.pry
```
Comment (LiannaNewman) [inline spec/controllers/api/v1/sandwiches_controller_spec.rb]: Don't forget to remove this `binding.pry`

**Review by LiannaNewman — CHANGES_REQUESTED** [review]

File: spec/controllers/api/v1/sandwiches_controller_spec.rb (hunk 1 above)
Comment (deniseileen) [reply spec/controllers/api/v1/sandwiches_controller_spec.rb]: I fixed this, adding a new pull request with the changes

**Review by LiannaNewman — APPROVED** [review]

**Comment by LiannaNewman** [comment]
Thanks for fixing those! This is good to go. Merging now.

**Pull request merged**